### PR TITLE
AP_Entity Default New Record Creating

### PR DESCRIPTION
**Pull request by AnastasiiaPV**

### CODE REVIEW

File: src/test/java/EntityDefaultTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+        WebElement saveButton = getDriver().findElement(By.xpath("//button[@id='pa-entity-form-save-btn']"));
+
+        js.executeScript("arguments[0].scrollIntoView();", saveButton);
+        saveButton.click();
+
+        List<WebElement> result = getDriver().findElements(By.xpath("//td[@class='pa-list-table-th']"));
+        for (int i = 0; i < 4; i++) {
+            Assert.assertEquals(result.get(i).getText(), expectedArray[i]);
+            System.out.println(result.get(i).getText());
```
Comment (OlenaKosenko) [inline src/test/java/EntityDefaultTest.java:78]: No need to print.

File: src/test/java/EntityDefaultTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        getDriver().findElement(By.xpath("//input[@id='t-11-r-1-date']")).click();
+        getDriver().findElement(By.xpath("//input[@id='t-11-r-1-datetime']")).click();
+
+        WebElement saveButton = getDriver().findElement(By.xpath("//button[@id='pa-entity-form-save-btn']"));
+
+        js.executeScript("arguments[0].scrollIntoView();", saveButton);
+        saveButton.click();
+
+        List<WebElement> result = getDriver().findElements(By.xpath("//td[@class='pa-list-table-th']"));
+        for (int i = 0; i < 4; i++) {
```
Comment (OlenaKosenko) [inline src/test/java/EntityDefaultTest.java:76]: 4 could be replaced by expectedArray length.

**Review by OlenaKosenko — CHANGES_REQUESTED** [review]

File: src/test/java/EntityDefaultTest.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        textTextField.sendKeys("new Text");
+        intTextField.sendKeys("25688");
+        decimalTextField.sendKeys("100.55");
+        dateField.click();
+        datetimeTextField.click();
+
+        WebElement plusButton = getDriver().findElement(By.xpath("//tr[@id='add-row-11']//button"));
+
+        JavascriptExecutor js = (JavascriptExecutor)getDriver();
+        js.executeScript("arguments[0].scrollIntoView();", plusButton);
```
Comment (Andrey2694) [inline src/test/java/EntityDefaultTest.java:49]: we have this method in utils just import it

File: src/test/java/EntityDefaultTest.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityDefaultTest.java]: All done, thank you!

File: src/test/java/EntityDefaultTest.java (hunk 2 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityDefaultTest.java]: I have done it. Thank you!

File: src/test/java/EntityDefaultTest.java (hunk 3 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityDefaultTest.java]: I have replaced it. Thank you!

**Review by OlenaKosenko — APPROVED** [review]

**Review by sviet-ka — APPROVED** [review]

**Pull request merged**